_guides/analytics/analytics_basics.md

```diff
@@ -0,0 +1,229 @@
+---
+layout: page
+title: Analytics&#58; the Basics
+order: 0
+---
+
+Start here to learn the basics about AMP analytics.
+
+{% include toc.html %}
+
+## Use amp-pixel or amp-analytics?
+
+AMP provides two components to meet your analytics and measurement needs:
+[amp-pixel](/docs/reference/amp-pixel.html) and
+[amp-analytics](/docs/reference/extended/amp-analytics.html).
+Both options send analytics data to a defined endpoint.
+
+If you are looking for behavior like a simple
+[tracking pixel](https://en.wikipedia.org/wiki/Web_beacon#Implementation),
+the `amp-pixel` component provides basic page view tracking;
+page view data gets sent to a defined URL.
+Some integrations with a vendor may call for this component,
+in which case they will specify the exact URL endpoint. 
+
+For most analytics solutions, use `amp-analytics`.
+Page view tracking works in `amp-analytics` too.
+But you can also track user engagement with any type of page content,
+including clicks on links, buttons, videos.
+And you can measure how far on the page the user scrolled,
+whether or not the user engaged with social media, and more.
+[Deep Dive into Amp Analytics](/docs/guides/analytics/deep_dive_analytics.html).
+
+As part of integrating with the AMP platform,
+many analytics providers have simplified `amp-analytics` configurations
+so that's it's easy to capture data and push to their tracking tools.
+Access vendor documentation from the
+[amp-analytics specification](/docs/reference/extended/amp-analytics.html).
```

Comment:
Yes, we should create a vendor doc, moving most of vendor content out of analytics spec. I've added this to my to-do list, and will create an issue for it.